_posts/2024-02-19-jco-1.0.md

```diff
@@ -0,0 +1,290 @@
+---
+title: "Announcing Jco 1.0"
+author: "Yosh Wuyts"
+date: "2024-02-21"
+github_name: "yoshuawuyts"
+excerpt_separator: <!--end_excerpt-->
+---
+
+We’re happy to announce the 1.0 release of [jco]: a native Javascript
+WebAssembly runtime and toolchain built for WebAssembly Components. Jco can
+natively run Wasm Components inside of Node.js, making it easier to take
+libraries written in different programming languages and execute them using the
+Node.js runtime. And by implementing [WASI
+0.2](https://bytecodealliance.org/articles/WASI-0.2), those components can
+access the network, filesystem, and other system APIs available in the Node.js
+runtime.
```

Comment:
Should we mention Preview 2 conformance - "with full conformance against the WASI 0.2.0 features`?

Reply:
I'm not sure how that is meaningfully different from what I've written here? What I wrote is: "implementing WASI 0.2" and linked to the WASI 0.2 post. You're suggesting we instead write: "full conformance against WASI 0.2 features".

I'm a little hesitant to use words like "conformance" if we don't also describe WASI 0.2 as a "specification". Again, as part of the in-group we understand the nuance of the WASI Subgroup vs the WebAssembly W3C WG's formal specifications. But given how careful we've been in our messaging so far, I'd prefer it if we stay clear from using words like "conformance" and stay closer to "we implement WASI 0.2" or "we provide WASI 0.2 interfaces", etc.

Reply:
I just added a footnote explaining what Wasm Components are vs what WASI 0.2 is. Hopefully that will help a little with this.

Reply:
The important point here I think is that we don't just implement "a bit of WASI" - but that we have spend months of engineering work ensuring we **fully support WASI**. That's a really key point to get across, that this is the whole thing, and it's fully stable. As opposed to being at the trailer of one sentence in the post.

Reply:
ok; changed the phrasing to match!